doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -369,6 +369,14 @@ Other samples
 
 |no_changes_yet_note|
 
+Device Tree Configuration
+=========================
+
+Thingy:91
+---------
+
+* Changed the PWM frequency of the pwmleds device from 50hz to 125hz
```

Comment:
```suggestion
* Changed the PWM frequency of the pwmleds device from 50 Hz to 125 Hz.
```
See https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Units+of+measurement